public/docs/ts/latest/guide/architecture.jade

```diff
@@ -53,14 +53,14 @@ figure
   Angular apps are modular and Angular has its own modularity system called _Angular Modules_ or _NgModules_.
```

Comment:
Got it. Will incorporate this throughout my work. Made Angular Modules ->Angular modules.